quick-start-nativescript/1-build-photo-album.md

```diff
@@ -69,22 +69,17 @@ The NativeScript companion app makes it easy for you to test your app on real de
 #### Action
 
 * **a.** Get your iOS or Android device.
-* **b.** Download and install the NativeScript companion app from the App Store or Google Play, respectively.
+* **b.** Download and install the Telerik Platform Companion app from your device's app store — i.e. the App Store for iOS users or Google Play for Android users (search for **Telerik Platform** or use the buttons provided below).
```

Comment:
The first result for “Telerik Platform” in the iOS App Store is the AppBuilder companion app. Perhaps use “Platform Companion” as that returns the more appropriate result first.